### PR TITLE
Feature media

### PR DESCRIPTION
**Pull request by asymetrique**
Player and zombie animation

### CODE REVIEW

File: player.go [hunk 1]
```diff
@@ -15,11 +15,64 @@ import (
...
+	Sprite        *SpriteSheet
+}
+
+// Update updates the state of the player
+func (p *Player) Update(g *Game) {
+	state := playerIdle
+
+	if ebiten.IsKeyPressed(ebiten.KeyW) {
+		p.MoveUp()
+		state = playerWalking
```
Comment (sinisterstuf) [inline player.go:39]: If you put this state change inside `player.move()` then you wouldn't have to write it out all 4 times, only once.

File: zombie.go [hunk 2]
```diff
@@ -16,9 +16,30 @@ import (
...
+	adjacent := z.Object.X - g.Player.Object.X
+	opposite := z.Object.Y - g.Player.Object.Y
+	z.Angle = math.Atan2(opposite, adjacent)
+
+	z.animate(g)
+	z.Object.Update()
+}
+
+func (z *Zombie) animate(g *Game) {
+	//No states at the moment, zombies are always walking
```
Comment (sinisterstuf) [inline zombie.go:35]: I'll add some soon, don't worry 😉

**Review by sinisterstuf — COMMENTED** [review]
I spotted a chance to simplify some state changes, the upside is you only write it once, the downside is code-wise it's not near where the other state change is, maybe we can find an even better place them later.

Apart from that looks good 👌

File: player.go (hunk 1 above)
Comment (asymetrique) [reply player.go]: Yep, I wasn't happy with it either. OK, I move it to move() for the time being

File: main.go [hunk 3]
```diff
@@ -90,11 +92,12 @@ func NewGame(g *Game) {
 
 	//Load sprites
 	g.Sprites = make(map[SpriteType]*SpriteSheet, 2)
-	g.Sprites[spritePlayer] = loadSprite("player")
-	g.Sprites[spriteZombie] = loadSprite("zombie")
+	g.Sprites[spritePlayer] = loadSprite("eZcort mission_Player Character")
+	g.Sprites[spriteZombie] = loadSprite("eZcort mission_Zombie")
```
Comment (sinisterstuf) [inline main.go:96]: Meh it would be cooler if we renamed the PNGs to nice short names.

**Review by sinisterstuf — APPROVED** [review]

File: main.go (hunk 3 above)
Comment (asymetrique) [reply main.go:96]: I asked Acros to use simple names, then we don't need to hassle with them always during a sprite update

**Pull request merged**